src/Pagination/Pagination.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, {css} from 'styled-components'
+import {get, COMMON} from '../constants'
+import theme from '../theme'
+import Box from '../Box'
+import {buildPaginationModel, buildComponentData} from './model'
+
+const Page = styled.a`
+  position: relative;
+  float: left;
+  padding: 7px 12px;
+  margin-left: -1px;
+  font-size: 13px;
```

Comment:
Should we move this to the theme too since it's different than the default 14px on most text?